packages/chord/index.d.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Return the available chord names
+ *
+ * @function
+ * @param {boolean} aliases - true to include aliases
+ * @return {Array} the chord names
+ *
+ * @example
+ * Chord.names() // => ["maj7", ...]
+ */
+export declare const names: () => string[];
+
+/**
+ * Get chord properties. It returns an object with:
+ *
+ * - name: the chord name
+ * - names: a list with all possible names (includes the current)
+ * - intervals: an array with the chord intervals
+ * - chroma:  chord croma (see pcset)
+ * - setnum: chord chroma number
+ *
+ * @function
+ * @param {String} name - the chord name (without tonic)
+ * @return {Object} an object with the properties or a object with all properties
+ * set to null if not valid chord name
+ */
+export declare const props: (str: string) => any;
```

Comment:
The return object has always the same shape. I liked when you add those types...